uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Upgrader.php

```diff
@@ -13,6 +13,7 @@ class CRM_HRLeaveAndAbsences_Upgrader extends CRM_HRLeaveAndAbsences_Upgrader_Ba
   use CRM_HRLeaveAndAbsences_Upgrader_Step_1005;
   use CRM_HRLeaveAndAbsences_Upgrader_Step_1006;
   use CRM_HRLeaveAndAbsences_Upgrader_Step_1007;
+  use CRM_HRLeaveAndAbsences_Upgrader_Step_1008;
```

Comment:
There's an upgrader with 1008 in Leave In Hours branch although it is yet to be merged to staging.
What do you think?

Reply:
I will keep it, whatever get merged first the other will have to fix the conflict in his branch 😄